.eslintrc.cjs

```diff
@@ -1,18 +1,23 @@
 /* eslint-env node */
+const process = require('process');
+const lintTypes = !!process.env.AGORICSDK_ESLINT_ENABLE_TYPE_RULES;
```

Comment:
I'd prefer an explicit test for `=== '1' || === 'true'` as the above would be truthy with `AGORICSDK_ESLINT_ENABLE_TYPE_RULES = '0'`

Reply:
I hear you prefer that. I don't think it's worth it given how this will be used. It's common for env flags to be based on presence, not the value.

Reply:
wfm. I considered CI but felt aversion because I wanted people to run it locally too, but pros:
1. it is a much more conventional flag
2. it often means "do more, slower stuff"
3. makes obvious when using it that it's the same settings as run in CI

Reply:
`CI` is really a conflation. I'm not really a fan of implicit behavior changes based on environment sniffing. As @turadg mentions, there is value in being able to explicitly run this test locally.

Reply:
I think this, combined with your other comment, is the best argument for a long-named environment variable (but please, `AGORIC_` prefix rather than `SMASHINGTHE_FIRST_TWO_WORDS_TOGETHER`), plus a script that runs with the appropriate NODE_OPTIONS and enabling environment variable.

Then CI can use that script directly as well.  No need to put it in `package.json`, I think, unless it's something like `lint-slow`.